concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1391,6 +1399,43 @@ jobs:
                   cf enable-feature-flag diego_docker
 
       - aggregate:
+        - task: deploy-compose-service-broker
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
+            inputs:
+              - name: paas-cf
+              - name: paas-compose-broker
+              - name: config
+              - name: bosh-CA
+            run:
+              path: sh
+              args:
+                - -e
+                - -u
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                  . ./config/config.sh
+                  echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+                  cf create-org service-brokers
+                  cf set-quota service-brokers small
```

Comment:
I think we should move the creation of this org out of this task. It might be better to make the existing `create-admin-org-space` task more generic to cover this org as well.